src/app/views/user/user-settings/user-settings.component.scss

```diff
@@ -39,68 +38,69 @@
         }
 
         .file {
-          width: 85px;
+          cursor: pointer;
           height: 85px;
           opacity: 0;
-          cursor: pointer;
+          width: 85px;
 
           &[disabled] {
             pointer-events: none;
           }
         }
 
         .icon-camera {
-          width: 85px;
+          bottom: 0;
+          display: table;
           height: 85px;
+          left: 0;
           pointer-events: none;
-          display: table;
           position: absolute;
-          top: 0;
-          bottom: 0;
-          left: 0;
           right: 0;
+          top: 0;
+          width: 85px;
 
           .fa-camera {
+            color: rgba(255, 255, 255, 0.7);
```

Comment:
`0.7` should be written without a leading zero as `.7`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)